cmd/bosun/sched/sched.go

```diff
@@ -540,6 +540,12 @@ const (
 
 func init() {
 	metadata.AddMetricMeta("bosun.statefile.size", metadata.Gauge, metadata.Bytes, "The total size of the bosun state file.")
+	metadata.AddMetricMeta("bosun.check.duration", metadata.Gauge, metadata.Second, "The number of seconds it took Bosun to check each alert rule.")
+	metadata.AddMetricMeta("bosun.check.err", metadata.Gauge, metadata.Error, "Running count of the number of errors Bosun has received while trying to evaluate an alert expression.")
+	metadata.AddMetricMeta("bosun.ping.resolved", metadata.Gauge, metadata.None, "1=Ping resolved to an IP Address. 0=Ping failed to resolve to an IP Address.")
+	metadata.AddMetricMeta("bosun.ping.rtt", metadata.Gauge, metadata.MilliSecond, "Number of milliseconds for the echo reply to be received. Also know as Round Trip Time.")
```

Comment:
know , known. Also complete sentences for these, ie "The running count..". Metadata should not be None for resolve, should be Bool or Ok.